### PR TITLE
Resolve possible ambiguities on vacancy filling

### PR DESCRIPTION
**Pull request by vrurg**
Define the rules of chosing a candidate for filling a vacancy

### CODE REVIEW

File: papers/Raku_Steering_Council_Code.md [hunk 1]
```diff
@@ -136,9 +136,15 @@ Whenever there is a vacancy during the regular council term, the
...
-them.
+If a council member drops out of touch and cannot be contacted for a month or
+longer, then the rest of the council may vote to replace them. If the vote is
+successful the new council member is chosen based on the last election results.
+The non-elected candidate with the most votes is offered with the position. If
+the offer is rejected then the next most-voted candidate is considered. 
+
+In case of a tie the rules of election come into effect.
+
+New election is started if no more candidates are left on the list.
```
Comment (JJ) [inline papers/Raku_Steering_Council_Code.md:148]: Better "*A* new election is sterted" of "will be started"

File: papers/Raku_Steering_Council_Code.md [hunk 2]
```diff
@@ -136,9 +136,15 @@ Whenever there is a vacancy during the regular council term, the
 council may vote to appoint a replacement to serve out the rest of the
 term.
 
-If a council member drops out of touch and cannot be contacted for a
-month or longer, then the rest of the council may vote to replace
-them.
+If a council member drops out of touch and cannot be contacted for a month or
+longer, then the rest of the council may vote to replace them. If the vote is
+successful the new council member is chosen based on the last election results.
+The non-elected candidate with the most votes is offered with the position. If
```
Comment (JJ) [inline papers/Raku_Steering_Council_Code.md:142]: scratch out *with*

File: papers/Raku_Steering_Council_Code.md [hunk 3]
```diff
@@ -136,9 +136,15 @@ Whenever there is a vacancy during the regular council term, the
...
+the offer is rejected then the next most-voted candidate is considered. 
+
+In case of a tie the rules of election come into effect.
```
Comment (JJ) [inline papers/Raku_Steering_Council_Code.md:145]: *will* come into effect.

File: papers/Raku_Steering_Council_Code.md [hunk 4]
```diff
@@ -136,9 +136,15 @@ Whenever there is a vacancy during the regular council term, the
 council may vote to appoint a replacement to serve out the rest of the
 term.
 
-If a council member drops out of touch and cannot be contacted for a
-month or longer, then the rest of the council may vote to replace
-them.
+If a council member drops out of touch and cannot be contacted for a month or
+longer, then the rest of the council may vote to replace them. If the vote is
+successful the new council member is chosen based on the last election results.
```
Comment (JJ) [inline papers/Raku_Steering_Council_Code.md:141]: *will be* chosen.

**Comment by vrurg** [comment]
@JJ thanks! 

Merging with the fixes.

**Pull request merged**